### DISCUSSION TITLE
Show all three editors

### DISCUSSION THREAD

**Discussion by thdoan** [FIameCaster/prism-code-editor#11] [Ideas]
What's the difference between minimalEditor, basicEditor, and fullEditor? It would be helpful to show a demo of all three editors (minimal, basic, full) on https://prism-code-editor.netlify.app/ so we can quickly see.

**Comment by jonpyt** [comment]
The website currently doesn't use any setups at all. The editors there are very similar to the full setup, but they also have the [`copyButton()`](https://prism-code-editor.netlify.app/api/functions/copy_button.copybutton) and [`highlightCurrentWord()`](https://prism-code-editor.netlify.app/api/functions/search.highlightcurrentword) extensions that the "full" setup doesn't have.

The [API documentation for the setups](https://prism-code-editor.netlify.app/api/modules/setups.html) tells you which extensions are added.

**Reply by thdoan** [reply]
Thanks for the link!
